.vscode/settings.json

```
@@ -0,0 +1,28 @@
+{
+    "python.linting.pylintEnabled": false,
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "eslint.workingDirectories": [
+        {
+            "directory": "src/vue",
+            "changeProcessCWD": true
+        }
+    ],
+    "python.pythonPath": "venv/bin/python",
+    "python.testing.pytestArgs": [
+        "src/django/",
+    ],
+    "python.analysis.extraPaths": [
+        "src/django"
+    ],
+    "python.analysis.stubPath": "",
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.envFile": "${workspaceFolder}/.env",
+    "python.formatting.provider": "black",
+    "editor.formatOnSave": true,
+    "python.formatting.blackArgs": [
+        "--line-length",
+        "120"
+    ],
```

```suggestion
    ],
    "cSpell.words": [
        "cleanup",
        "conn",
        "jirs",
        "resp"
    ],
```

Something we add to together for this workspace